duckduckgo.safariextension/html/global.html

```diff
@@ -219,7 +248,15 @@
             }
         }, false);
     }
-
+    
+    safari.application.addEventListener('navigate', function(evt) {
```

Comment:
Should we add a guard to check `window.SafariNavigateEvent` exists so we don't break old Safari users? Seems like this should work in versions 5.1+. I'm not sure if we have anyone using it on older versions, but we guard against the other event above: https://github.com/duckduckgo/safari-zeroclickinfo/pull/19/files#diff-89d925e2bf34fb6e3d713637cfd75bdbR196